wtransport/src/tls.rs

```diff
@@ -31,7 +31,7 @@ pub enum CertificateLoadError {
 pub struct InvalidCertificate(usize);
 
 /// A server TLS certificate.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
I am not totally sure about that.

While it is Ok having a `Debug` printing the `certificates` chain, this change will do the same for `private_key`, and possibly leaking very sensible information by accident.

What is your use case for this? 
If that's just for checking certificate, I am wondering if it could be enough the API `.certificates()` (which you can directly print like `println!({:?}, cert.certificates())`), or, even better `.hashes()`, which returns a fingerprint of the certificates (which now implements handy way to print even in human-readable form)

Also remember `private_key` is always accessible via getter, but having a dedicate (and fully named) method makes it having more intentional.